### ISSUE TITLE
Tablet resize is breaking responsiveness

### ISSUE THREAD

**Issue by scottdf** [newaeonweb/ResponsiveAeon-Cssgrid#1]
There is an issue when you rotate your framework on a tablet. The css will show correctly for narrow viewport but when the tablet (ipad specifically) is rotated wide the code does not resize itself correctly. It's too wide and forces the user to shrink the site to fit on page. I have noticed this issue on your demo as well.

**Comment by newaeonweb** [comment]
Hello Scottthanx for the info, we registered this bug and provide the fix
asap!!
thanx.
Fernando Monteiro
www.newaeonweb.com.br
fernando@newaeonweb.com.br
www.twitter.com/newaeonweb
+55 (11) 9-5023-5330
skype: fernandochc

---- Original Message ----
From: scottdf 
To: "newaeonweb/ResponsiveAeon-Cssgrid" 
Sent: Ter, Nov 27, 2012, 22:06 PM
Subject: [ResponsiveAeon-Cssgrid] Tablet resize is breaking responsiveness
(#1)
    There is an issue when you rotate your framework on a tablet. The css will
show correctly for narrow viewport but when the tablet (ipad specifically)
is rotated wide the code does not resize itself correctly. It's too wide
and forces the user to shrink the site to fit on page. I have noticed this
issue on your demo as well.            

```
              —
          Reply to this email directly or view it on GitHub
```

(https://github.com/newaeonweb/ResponsiveAeon-Cssgrid/issues/1).

**Comment by scottdf** [comment]
Hey Fernando,

First off, I love your work. This was my first time working with a 100% responsive website and your framework has made it loads easier. I ended up closing the issue but I noticed the fix was interesting. I believe it all has to do with the meta viewport and user scalable. Using your new settings the ‘reload’ doesn’t happen when adjust the tablet, however the ability to pinch zoom is gone. Looking into the Twitter Bootstrap I’m seeing the same issues. I think this is just an issue with using the meta viewport.

What did you find?
